src/common/localization/en-us.ts

```diff
@@ -272,6 +272,7 @@ export const english: IAppStrings = {
         messages: {
             saveSuccess: "Successfully saved ${connection.name}",
             deleteSuccess: "Successfully deleted ${connection.name}",
+            doNotAllowDuplicateNames: "Do not allow duplicate names"
```

Comment:
"Do not allow duplicate names" doesn't make sense for the user.

Something like "Connection with name \"${connectionName}\" already exists. Please, use another name" is better